ledger/appcow.go

```diff
@@ -519,6 +532,17 @@ func (cb *roundCowState) BuildEvalDelta(aidx basics.AppIndex, txn *transactions.
 			}
 		}
 	}
+
+	// logDeltas
+	for appid, ldelta := range cb.logs {
```

Comment:
This makes me think `cb.log` should just be an array too, rather than map.  But I'm not 100% sure.  Perhaps we want to use this "wider" representation in ApplyDelta in preparation for differentiating log messages from multiple apps, once/when we allow apps to call apps.

Reply:
for now I will use an array instead of a map.